net-misc/openssh/openssh-7.7_p1-r7.ebuild

```diff
@@ -0,0 +1,443 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Your commit message summary line is too long. I don't get why repoman permitted you to do that.

Reply:
On purpose. The chosen length restriction is an arbitrary value and I believe in usability about rules like that. However, I'll drop "rev bump to " and then it should fit.

Reply:
The existence of corner cases where the policy doesn't exactly work **does not justify ignoring it in cases it does work**. Yes, I wouldn't complain if you couldn't fit a commit message in the limit because the package name was impossibly long. Of course, we could argue that using package names this long **defeats readability** itself.

But when you can fit the commit message within limits without any problem, and you purposely ignore the policy because of sheer laziness or arrogance, that's plain wrong. This attitude is plain wrong for a Gentoo developer, and even more wrong for a Council member. What example does a Council member set if he blatantly chooses to ignore policies set by the Council? Even if you weren't at the Council at the moment, or if you voted 'no', those policies apply to you too.

Reply:
>The existence of corner cases where the policy doesn't exactly work does not justify ignoring it in cases it does work.

I agree with this and that's why I said in my first comment

>I'll drop "rev bump to " and then it should fit

...and I did this so this should be resolved.

I also agree with

> purposely ignore the policy because of sheer laziness or arrogance, that's plain wrong

However I am not doing it just because I don't *like*. I think I have reasons (sometimes). Of course, one could argue that this is already a part of the problem if individuals belief they know something better and take that as excuse to act against rules... but... well, I hope my examples were clear and you get my point like I got yours. Let's move on.